_data/governance.yml

```diff
@@ -0,0 +1,37 @@
+introText: This section is updated every 6 months by the Scala Center.
+cards:
+  - title: "Scala Improvement Process"
+    description: "The SIP is the primary mechanism for evolving the Scala language."
+    expandText: "Process & People"
+    links:
+      - name: SIP Committee Members
+        url: https://docs.scala-lang.org/sips/process-specification.html#the-sip-committee
+      - name: SIP Process Specification
+        url: https://docs.scala-lang.org/sips/process-specification.html
+  - title: "Scala 3 Maintenance"
+    description: "Managing the Scala 3 compiler, standard library and documentation."
+    expandText: "Process & People"
+    links:
+      - name: Core Maintainers
+        url: https://github.com/lampepfl/dotty/blob/main/MAINTENANCE.md#compiler
+      - name: Maintenance Procedures
+        url: https://github.com/lampepfl/dotty/blob/main/MAINTENANCE.md
```

Comment:
Here too, both links point to the same page. Also, that page dives deep into the issue supervisor role without giving an overview of all the organizations (I see at least LAMP and VirtusLab) involved in the maintenance of the compiler and their respective role.

Last, the description above mentions the standard library and the documentation, which are not covered by the links.

Reply:
This information should probably be added to the MAINTENANCE.md as a separate PR.